servers/services/src/main/java/org/projectnessie/services/cel/CELUtil.java

```diff
@@ -45,6 +45,8 @@ public final class CELUtil {
   public static final String VAR_REF_META = "refMeta";
   public static final String VAR_COMMIT = "commit";
   public static final String VAR_ENTRY = "entry";
+  public static final String VAR_CONTENT_KEY = "contentKey";
+  public static final String VAR_NAME = "name";
```

Comment:
Do we need to add them to `ENTRIES_DECLARATIONS`? Line 70

```
public static final List<Decl> ENTRIES_DECLARATIONS =
      ImmutableList.of(
          Decls.newVar(VAR_ENTRY, Decls.newMapType(Decls.String, Decls.String)),
          Decls.newVar(VAR_NAMESPACE, Decls.String),
          Decls.newVar(VAR_CONTENT_TYPE, Decls.String));
```

I know testcases are passing. I need to dig more to understand the implications if we don't add them to declarations.

Reply:
refactored.